dm/templates/iam_member/iam_member.py.schema

```diff
@@ -15,7 +15,16 @@
 info:
   title: IAM policy member
   author: Sourced Group Inc.
-  description: Manages an IAM policy member
+  version: 1.0.0
+  description: |
+    Manages an IAM policy member
+
+    For more information on this resource:
+    https://cloud.google.com/iam/docs/overview
+
+    APIs endpoints used by this template:
+    - gcp-types/cloudresourcemanager-v1:virtual.projects.iamMemberBinding =>
+        TODO
```

Comment:
What does TODO mean here? Documentation of the virtual type?

Reply:
Yes